src/pytest_cpp/catch2.py

```diff
@@ -20,11 +21,11 @@ class Catch2Facade(AbstractFacade):
     """
 
     @classmethod
-    def is_test_suite(
+    def get_catch_version(
         cls,
         executable: str,
         harness_collect: Sequence[str] = (),
-    ) -> bool:
+    ) -> Optional[str]:
```

Comment:
If we want to type this correctly, we can either use an enum, or `Literal`:

```suggestion
    ) -> Optional[Literal["v2", "v3"]]:
```